test/cypress/integration/group-settings.spec.js

```diff
@@ -119,15 +119,6 @@ describe('Group Voting Rules', () => {
     })
   }
 
-  it('user1 creates a group with rule "disagreement" of "2"', () => {
-    cy.giCreateGroup('groupDis_2', { ruleName: 'disagreement', ruleThreshold: 2 })
-
-    verifyRuleSelected('disagreement', {
-      status: '2',
-      ruleAdjusted: false
-    })
-  })
-
```

Comment:
Could you either comment this out or add `it.skip` to the test?

We don't want to remove support for it completely. Just for now and just in the group creation step. We might allow users to switch to it manually themselves.

Reply:
done it in this [commit](https://github.com/okTurtles/group-income/pull/1407/commits/1478f02b565169320ea860dc549665f64aff2401)